controllers/nodefeaturediscovery_resources.go

```diff
@@ -69,34 +82,65 @@ func filePathWalkDir(root string) ([]string, error) {
 	return files, err
 }
 
+// getAssetsFrom takes a path as an input and grabs all of the
+// file names in that path, then returns a list of the manifests
+// it found in that path.
 func getAssetsFrom(path string) []assetsFromFile {
 
+	// manifests is a list type where each item in the list
+	// contains the contents of a given asset (manifest)
```

Comment:
Much of that is self-evident from the code

```suggestion
// All assets (manifests) as raw data
```